notebooks/3_KernelMethods.ipynb

```diff
@@ -35,9 +35,13 @@
     "from utilities.plotting import (\n",
     "    plot_projection, plot_regression, check_mirrors, get_cmaps, table_from_dict\n",
     ")\n",
-    "from utilities.kernels import linear_kernel, gaussian_kernel, center_kernel\n",
-    "from utilities.classes import PCA, LR, KPCA, KRR, KPCovR\n",
-    "\n",
+    "from utilities.kernels import linear_kernel, gaussian_kernel,center_kernel # to be replaced with scalers from skcosmo\n",
+    "#from sklearn.metrics.pairwise import linear_kernel, rbf_kernel \n",
+    "#I found that the result of rbf_kernel(gamma = 1.0) in sklearn is not the same as that of gaussian_kernel(gamma = 1.0) in utilities, so I keep the original gaussian_kernel.\n",
```

Comment:
Are you properly centering and scaling? Please figure this out before we merge, we need to use sklearn where possible.

Reply:
I don't think it has anything to do with centering and scaling. It seems to be caused by a numerical error and should be related to the implementation of gaussian_kernel in utilities.
![image](https://user-images.githubusercontent.com/56018055/99538480-51991500-29ad-11eb-836c-961f66a4b43c.png)

Reply:
Can you print a `np.linalg.norm(K_train_sklearn - K_train_utility)`? This looks like a simple rounding error.

Reply:
![image](https://user-images.githubusercontent.com/56018055/99546571-d5a3ca80-29b6-11eb-895e-48abdf265039.png)

Reply:
Exactly, that error is so small that it means it's a precision error, and they give the exact same result. Please edit accordingly.

Reply:
Okay.